doc/api/errors.md

```diff
@@ -1886,3 +1892,4 @@ Creation of a [`zlib`][] object failed due to incorrect configuration.
 [try-catch]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/try...catch
 [vm]: vm.html
 [WHATWG Supported Encodings]: util.html#util_whatwg_supported_encodings
+[`require`]: modules.html#modules_require
```

Comment:
1. It seems this should be:
```md
[`require()`]: modules.html#modules_require
```
2. We usually sort bottom references ASCII-wise, so this need to be placed after the `` [`require('crypto').setEngine()`]: `` item.